pkg/servicemirror/util.go

```diff
@@ -9,16 +9,19 @@ import (
 
 // WatchedClusterConfig contains the needed data to identify a remote cluster
 type WatchedClusterConfig struct {
-	APIConfig     []byte
-	ClusterName   string
-	ClusterDomain string
+	APIConfig        []byte
+	ClusterName      string
+	ClusterDomain    string
+	LinkerdNamespace string
 }
 
 // ParseRemoteClusterSecret extracts the credentials used to access the remote cluster
 func ParseRemoteClusterSecret(secret *corev1.Secret) (*WatchedClusterConfig, error) {
 	clusterName, hasClusterName := secret.Annotations[consts.RemoteClusterNameLabel]
 	config, hasConfig := secret.Data[consts.ConfigKeyName]
 	domain, hasDomain := secret.Annotations[consts.RemoteClusterDomainAnnotation]
+	ld5Namespace, hasLd5Namespace := secret.Annotations[consts.RemoteClusterLinkerdNamespaceAnnotation]
```

Comment:
```suggested
l5dNamespace, hasL5dNamespace := secret.Annotations[consts.RemoteClusterLinkerdNamespaceAnnotation]
```

Reply:
ooops

Reply:
Ah it's `'''suggestion` not `'''suggested`.. but yes you see what I'm pointing out 😆